### ISSUE TITLE
ImgBot Not Generating PR

### ISSUE THREAD

**Issue by danner26** [imgbot/Imgbot#1342]
Hello, this isn't as much of an issue as it is asking for troubleshooting help. It might end up being an issue but first I need to figure out how to debug this. I have a private repo that has a lot of images in it (over 2000 and growing). I have imgbot enabled on it and I can request an optimization within the imgbot.net website.

The issue that I am having is that I do not see a PR being created, and I do not know how to track down if A) the images are fine and don't need to be updated, or B) if there is an error/too many images to process. Can anyone assist in troubleshooting this?

**Comment by MindInSky** [comment]
Bumping this

This is happening to me too on a public repo, however I'm using the student pack, not sure if related

**Comment by Ubuntufoo** [comment]
Same, public repo, student pack, around 25 images. Stopped working a month ago. I removed the app from Github applications, reconnected it via the student offer link, and have requested fresh optimization via https://imgbot.net/app/ but no PR ever arrives for any repo.

**Comment by MindInSky** [comment]
It just seems to be very slow at the initial repos, maybe due to a queue? If this is the case a heads up would be more than enough

**Comment by Gitstar-OC** [comment]
No, I think this bot servers are down and no one is working to solve this, maybe any alternative to this must be created.

**Comment by JosefHobler** [comment]
I have the same issue. Student pack and no PR for last 2 or more months.
